### PR TITLE
chore(RELEASE-1038): add embargoed-cves internal task and pipeline

### PR DESCRIPTION
**Pull request by johnbieren**
This commit moves the embargoed-cves pipeline and task from the app-interface repo to the internal directory of this repo. It also adds tests and a README with it.

### CODE REVIEW

File: internal/tasks/check-embargoed-cves-task/README.md
Comment (mmalina) [inline internal/tasks/check-embargoed-cves-task/README.md:1]: Can you also add changelog entries for all the versions until now? Just kidding :D

But one side note: I understand the reasoning behind making it always succeed, but it sucks, because then you cannot easily see what failed in the internal-services cluster console. (Like when debugging failing signing pipelines - in that case they really do fail.)

Nothing to do here, just saying.

**Review by mmalina — APPROVED** [review]

File: internal/tasks/check-embargoed-cves-task/README.md
Comment (johnbieren) [reply internal/tasks/check-embargoed-cves-task/README.md:1]: Lol

Yeah, agreed. Hopefully one day we can remove internal requests and then we won't have that issue, but until then I don't think we can really do much about it

**Review by happybhati — APPROVED** [review]

**Comment by johnbieren** [comment]
Should merge after https://github.com/konflux-ci/release-service-catalog/pull/643

**Review by happybhati — APPROVED** [review]

**Comment by johnbieren** [comment]
Nothing to do with E2E so merging without it since it is broken anyways.
But won't merge until approval on https://github.com/hacbs-release/app-interface-deployments/pull/211

**Pull request merged**
